Identification-and-Assessment.tex

```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
-	\item Plan Type, and a minimum of one related Reponse Plan
-	\item Realized Risk Plan --- Description of event trigger
+	\item Cost and/or Schedule Impact Description
+	% if over certain severity. PJM: I think we always want a brief description of any impact we specify.
+	\item Cost and/or Schedule fields under Analyze Risk Quantitative, if impact is non-negligible.
+	The Likely Cost is the estimated cost of addressing the risk in the year after it is realized.
+	(All these short-term cost estimates are used to forecast the required annual level of reserve funds.)
+	The Likely Delay is the estimated length of delay to the Impacted Milestone.
+	If the Impacted Milestone is either ``Start of Operations'' or ``LSST Survey Completion,'' the Realized Risk Plan (see below) needs to include ``Extend Survey.''
+	(Such schedule delays incur a post-ops cost.)
+	\item Plan Type, and a minimum of one related Response Plan (i.e. how the risk will be mitigated)
+	\item Realized Risk Plan --- Description of event trigger, and the plan for how the risk will be addressed once it is realized.
 	\item All fields under Analyze Risk Quantitative --- note that these do not automatically populate any required fields.
 	\item Existential Risk
 \end{itemize}
 
 Optional fields:
 \begin{itemize}
 	\item Overall Impact --- Impact category that can override the automatically generated cost and schedule impact categories (sets to most severe)
-	\item Cost or Schedule Impact Description
+	% \item Cost or Schedule Impact Description. PJM: See above, I think we always need a description any time we specify an impact.
 	\item Related Actions
 \end{itemize}
 
 Verify assessment in the automatically generated fields under Analyze Risk Score.
 
+When analyzing a risk that is the parent of $N$ child risks:
+\begin{itemize}
+	\item First analyze the children separately, i.e. estimating Residual Probability $P_k$, Likely Cost $C_k$, and Likely Delay $S_k$ for the $k$-th risk, and repeating for all $k$ in the range $(1 \ldots N)$.
+	\item For each risk, calculate the Cost Exposure $E_k = C_k \times P_k$.
+	\item The combined Cost Exposure for the parent risk is just the sum of the Cost Exposures of its children, $E_{\rm total} = \sum_{k=1}^N E_k$.
+	\item The Likely Cost of the parent risk is the sum of the Likely Costs of its children, $C_{\rm total} = \sum_{k=1}^N C_k$.
+	\item The Residual Probability of the parent risk is its Cost Exposure divided by its Likely Cost, $P_{\rm total} = E_{\rm total} / C_{\rm total}$.
+	This is just the cost-weighted average of the child risks' Residual Probabilities, and is the most meaningful definition of probability for parent risks.
+	\item Other quantitative attributes of parent risks can be computed by simple summation (e.g. Minimum Cost, Maximum Delay, Schedule Exposure, etc).
+	\item Pre-response quantities of parent risks, such as Impact Scores, can be reverse engineered from their Residual Probabilities.
+\end{itemize}
```

Comment:
Same argument for most of these child risks as for Cost Exposure. Perhaps we could say something like ... "To first order, the estimated Cost Exposure of the parent risk is the sum of the Cost Exposure of its children". Ditto for all other parameters